### PR TITLE
Update to new file_io messages

### PR DESCRIPTION
**Pull request by mfine**
Pick up the new file_io messages. See https://github.com/swift-nav/libsbp/pull/183.

/cc @fnoble @mookerji @denniszollo

### CODE REVIEW

File: piksi_tools/fileio.py [hunk 1]
```diff
@@ -38,13 +39,14 @@ def read(self, filename):
     buf = ''
     while True:
       msg = struct.pack("<IB", len(buf), chunksize) + filename + '\0'
-      self.link.send(SBP_MSG_FILEIO_READ, msg)
-      data = self.link.wait(SBP_MSG_FILEIO_READ, timeout=1.0)
+      self.link.send(SBP_MSG_FILEIO_READ_REQUEST, msg)
+      data = self.link.wait([SBP_MSG_FILEIO_READ_REQUEST,
```
Comment (mookerji) [inline piksi_tools/fileio.py]: `wait` only dispatches on a single msg ID, right? Would you want to do ....

```python
data = self.link.wait(SBP_MSG_FILEIO_READ_RESPONSE, timeout=1.0)
```

Comment (mfine) [inline piksi_tools/fileio.py]: You can wait on multiple messages... one of the benefits that came out of https://github.com/swift-nav/libsbp/pull/174.

Comment (mookerji) [inline piksi_tools/fileio.py]: I totally missed that.

File: piksi_tools/fileio.py [hunk 2]
```diff
@@ -66,13 +68,14 @@ def readdir(self, dirname='.'):
     files = []
     while True:
       msg = struct.pack("<I", len(files)) + dirname + '\0'
-      self.link.send(SBP_MSG_FILEIO_READ_DIR, msg)
-      data = self.link.wait(SBP_MSG_FILEIO_READ_DIR, timeout=1.0)
+      self.link.send(SBP_MSG_FILEIO_READ_DIR_REQUEST, msg)
```
Comment (mookerji) [inline piksi_tools/fileio.py]: Is `SBP_MSG_FILEIO_READ_DIR_REQUEST` in a state where we can actually construct an object and and send that over, or is it still broken w.r.t. Construct?

Comment (mfine) [inline piksi_tools/fileio.py]: It was broken.

Comment (mookerji) [inline piksi_tools/fileio.py]: *sigh*

**Comment by mookerji** [comment]
Looks good to me!

**Pull request merged**